docs/_integration-manager/camel-routes/camel-route-aws.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: Camel Route Configuration for AWS S3
+nav_order: 2
+parent: Camel Route Configuration
+---
+# Camel Route Configuration for AWS S3
+
+## Overview
+
```

Comment:
## Overview

Monitor an S3 folder for new files and submit to the JobConfig run API

Reply:
Done.